test/integration/features/basic.feature

```diff
@@ -35,16 +42,25 @@ Feature: Basic test
         And stdout should contain "Setting up KVM"
         And stdout should contain "Installing libvirt service and dependencies" 
         And stdout should contain "Adding user to libvirt group"
+        And stdout should contain "Will use root access: add user to libvirt group"
         And stdout should contain "Enabling libvirt"
         And stdout should contain "Starting libvirt service"
+        And stdout should contain "Will use root access: start libvirtd service"
         And stdout should contain "Installing crc-driver-libvirt"
         And stdout should contain "Removing older system-wide crc-driver-libvirt"
         And stdout should contain "Setting up libvirt 'crc' network"
         And stdout should contain "Starting libvirt 'crc' network"
         And stdout should contain "Checking if NetworkManager is installed"
         And stdout should contain "Checking if NetworkManager service is running"
         And stdout should contain "Writing Network Manager config for crc"
+        And stdout should contain "Will use root access: write NetworkManager config in /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf"
+        And stdout should contain "Will use root access: execute systemctl daemon-reload command"
+        And stdout should contain "Will use root access: execute systemctl stop/start command"
         And stdout should contain "Writing dnsmasq config for crc"
+        And stdout should contain "Will use root access: write dnsmasq configuration in /etc/NetworkManager/dnsmasq.d/crc.conf"
+        And stdout should contain "Will use root access: execute systemctl daemon-reload command"
+        And stdout should contain "Will use root access: execute systemctl stop/start command"
+        And stdout should contain "Unpacking bundle from the CRC binary"
         And stdout should contain "Setup is complete"
```

Comment:
Up to this point, none of the changes in this commit  seem mac-related?

Reply:
@cfergeau yes, that should be correct. Why is it bad?

Reply:
They are part of a commit whose log is ` extra: changed virtualbox to hyperkit in crc start for mac with -d...` so it's not really clear why they are in this commit, what they are going to achieve, ...

Reply:
@cfergeau updated & split into 3 commits topic-wise. Thanks for noticing